azure/functions_worker/bindings/eventhub.py

```diff
@@ -93,5 +93,8 @@ def from_proto(cls, data: protos.TypedData, *,
             sequence_number=cls._decode_trigger_metadata_field(
                 trigger_metadata, 'SequenceNumber', python_type=int),
             offset=cls._decode_trigger_metadata_field(
-                trigger_metadata, 'Offset', python_type=str)
+                trigger_metadata, 'Offset', python_type=str),
+            device_id=cls._decode_trigger_metadata_field(
```

Comment:
IOT trigger sends other metadata as well:
```
  "iothub-connection-device-id": "miye-device0",
  "iothub-connection-auth-method": "{\"scope\":\"device\",\"type\":\"sas\",\"issuer\":\"iothub\",\"acceptingIpFilterRule\":null}",
  "iothub-connection-auth-generation-id": "636640121350566756",
  "iothub-enqueuedtime": "2018-12-13T14:42:43.358Z",
  "iothub-message-source": "Telemetry",
```
We should add all of them

Reply:
The metadata already has an `enqueued time` property.  Should we prefix all of these with `iothub` to disambiguate?

Reply:
We need both. For metadata for eventhub is enqueued  time for IOT hub it is iothub-enqueuedtime.
Note: In case of IOT hub - the values are the same. enqueued  time can be accessed via `enqueued ` or `iothub-enqueuedtime`